tsconfig.json

```diff
@@ -14,7 +14,9 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "baseUrl": "./",
+    "paths": { "*": ["declarations/*"] }
```

Comment:
Path rewriting can get pretty messy easily. Is this even being used?

Reply:
good catch, tried to fix the new libraries but ended up fixing it properly instead of this, removing